### DISCUSSION TITLE
We mistakenly set a personal profile to an organization.

### DISCUSSION THREAD

**Discussion by AraratChess** [community/community#112417] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

We mistakenly set a personal profile to an organization. Now we want to change it back but we can't, please help us to solve this problem!!

**Comment by ghostinhershell** [comment]
Hey there!  👋

Thanks for posting in the GitHub Community, @AraratChess! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

With that being said I do want to note a couple of things:
1. I've removed your private information from this post as posting identifiable information is not allowed, per our [Code of Conduct](https://github.com/community/community/blob/main/CODE_OF_CONDUCT.md).
2. The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :)

I've gone ahead and moved this to the correct category for you. Good luck!
